### ISSUE TITLE
A question about the cascade cnn

### ISSUE THREAD

**Issue by MaxLingwei** [anson0910/CNN_face_detection#28]
Dear anson,
I'm sorry for trouble you again. I find that in your code, the part of detecting face, your 24 net just use the result of 12net directly. But in the paper I find that the 24net and 12net are only connected in fully-connected layer, so as 48net and 24net.
So please tell me whether I have a wrong understanding if convient.
Thank you for your help!

**Comment by anson0910** [comment]
Hi,
In the paper, 12 net is followed by 24 net and then 48 net.
I think you are referring to the multi-resolution architectures of 24 net and 48 net.

If you wish to do so,
I believe this can be done by the following steps:
1. Add the 12-net layers to the 24-net .prototxt files (concatenating the fully connected layers as shown in the paper)
2. Transplant the trained 12-net parameters to the multi-resolution net, and set the local learning rate to 0, as shown [here](https://groups.google.com/forum/#!msg/caffe-users/wfwl468OpG4/JlnnCcaEboYJ)
3. Train the net

**Comment by MaxLingwei** [comment]
I see. It seems that I have a wrong understanding for the paper before.
Thank you for your answering!
